gen/AArch64Arch_gen.ml

```diff
@@ -797,7 +797,7 @@ let sequence_dp (d1,c1) (d2,c2) = match c1 with
 let expand_dp_dir (dir,_) = D.expand_dp_dir dir
 
 (* Read-Modify-Write *)
-type rmw =  LrSc | LdOp of atomic_op | StOp of atomic_op | Swp | Cas
+type rmw =  LrSc | LdOp of atomic_op | StOp of atomic_op | Swp | Cas | AllAmo
```

Comment:
I am not sure I understand this. What's the point? `Amo` appears as a possible in diyone7 and when I tried I got:
```
$> diyone7 -arch AArch64 Amo Rfe PosRW Rfe -oneloc
Fatal error: exception File "gen/AArch64Arch_gen.ml", line 896, characters 16-22: Assertion failed
```
If I understand correctly the intent is for diy7 to go through all possible Amo, isn't it? Is there a way for this to not appear. as a valid relaxation in diyone7?

Reply:
And by the way this seems to be the case for `Dp**` and I guess many others, some of these shouldn't appear as relaxation options for diyone7.